### PR TITLE
docs: review crud and root help messages

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: messages/delete/messages.go [hunk 1]
```diff
@@ -3,6 +3,6 @@ package delete
 var (
 	Usage            = "delete"
 	ShortDescription = "Deletes a resource"
-	LongDescription  = "Deletes a resource based on the given identification information"
+	LongDescription  = "Deletes a resource based on the given ID information"
```
Comment (PatrickMenoti) [inline messages/delete/messages.go:6]: ID information? 🤔

File: messages/delete/rules_engine/messages.go [hunk 2]
```diff
@@ -1,14 +1,14 @@
 package rulesengine
 
 var (
-	AskInputRulesId       = "What is the id of the Rule Engine you wish to delete?"
-	AskInputApplicationId = "What is the id of the Edge Application this Rule Engine is linked to?"
-	AskInputPhase         = "What is the phase of your rule engine? (request/response)"
+	AskInputRulesId       = "What's the id of the rules engine you wish to delete?"
```
Comment (PatrickMenoti) [inline messages/delete/rules_engine/messages.go:4]: very casual, huh

Comment (gabriel-azion) [reply messages/delete/rules_engine/messages.go:4]: a little friendlier to the end user

**Review by PatrickMenoti — APPROVED** [review]
Nice!

**Pull request merged**